### ISSUE TITLE
Unable to scroll editor content

### ISSUE THREAD

**Issue by avinashyalgonde** [imnapo/react-native-cn-quill#41]
@imnapo  I am unable to scroll editor content after adding multiple lines. Is there any way to scroll content?

**Comment by imnapo** [comment]
yes, It should scroll the content. I've checked the example project and it's working as expected. Please post your code here to be able to help you.

**Comment by swapnilgaikwad1507** [comment]
In Android, I am able to scroll when keyboard is closed. But as soon as I open keyboard editor stops scrolling.

https://user-images.githubusercontent.com/47855902/114850407-5c41c400-9dfe-11eb-9225-050cb0c84c82.mp4

**Comment by avinashyalgonde** [comment]
@swapnilgaikwad1507 same for me. Let me know if you find any solution

**Comment by imnapo** [comment]
unfortunately, Recently I'm unable to run expo app on my device anymore. I'll try to fix it as soon as possible.

**Comment by EstebanIndiveri** [comment]
same problem in "react-native-cn-quill": "^0.7.8" unable scroll input multiline android only with select

**Comment by AustinWood** [comment]
if quill editor is inside scrollview, quill editor scroll not working for long text.
it's not working in android only - in ios, it performs well.

**Comment by AustinWood** [comment]
quill editor is embedded in webview and it is known bug of react native that webview is not scrollable inside scrollview.
if someone encountered this problem, please let me know solution. Thanks!

**Comment by AustinWood** [comment]
this issue can be fixed by setting **nestedScrollEnabled** to true in **_renderWebview_** function of **quill-editor.tsx**. 
and also, set **scrollEnabled** props to true and simply run **_'npx patch-package react-native-cn-quill'_** to make patch.

**nestedScrollEnabled** props is newly defined in recent react-native-webview so you would need to upgrade that module first if it's old.

**Comment by imnapo** [comment]
You may pass any property of `react-native-webview` to `QuilEditor` like below:
``` 
<QuillEditor
          webview={{
            scrollEnabled: true,
            nestedScrollEnabled: true,
          }}
/>
```

**Comment by HimikoT-T** [comment]
> You may pass any property of `react-native-webview` to `QuilEditor` like below:
> 
> ```
> <QuillEditor
>           webview={{
>             scrollEnabled: true,
>             nestedScrollEnabled: true,
>           }}
> />
> ```

It worked, but the toolbar is not showing anymore after scrolling down

**Comment by JacobDel** [comment]
Were you able to fix it @tylerrhinehart ?

**Comment by tylerrhinehart** [comment]
@JacobDel I was not able to fix it. After working on this for two solid days, I decided it wasn't worth any more time. I ended up keeping it simple and having a fixed height.
